libraries/cmake/source/krabsetw/config/krabs.cpp

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+// Disable warning for KrabsETW code
```

Comment:
It would be helpful to comment why this warning is okay to disable in this context.

Reply:
this is not required anymore so I've removed it